shell/platform/embedder/embedder_external_texture_gl.h

```diff
@@ -27,7 +27,10 @@ class EmbedderExternalTextureGL : public flow::Texture {
   sk_sp<SkImage> last_image_;
 
   // |flow::Texture|
-  void Paint(SkCanvas& canvas, const SkRect& bounds, bool freeze) override;
+  void Paint(SkCanvas& canvas,
+             const SkRect& bounds,
+             bool freeze,
+             GrContext* context) override;
```

Comment:
I think we should avoid adding this.  It would be better if we could make sure the `canvas` has the right context.